seahorse-workflow-executor/deeplang/src/main/scala/ai/deepsense/deeplang/DOperation.scala

```diff
@@ -98,9 +102,39 @@ abstract class DOperation extends Operation
   def inferGraphKnowledgeForInnerWorkflow(context: InferContext): GraphKnowledge = GraphKnowledge()
 
   def typeTag[T : ru.TypeTag]: ru.TypeTag[T] = ru.typeTag[T]
+
+  val reportType: ChoiceParam[ReportType] = ChoiceParam[ReportType](
+    name = "report type",
+    description = Some(""))
```

Comment:
why empty description? This is something that gets rendered in the tooltip. Should be something informative, not empty